### PR TITLE
Room Attendant and Authentication changes after fixing merge conflicts from latest main

### PR DESCRIPTION
**Pull request by sammu1991**
After starting the java application.

Run this sql statement in MySql --

INSERT INTO roomranger.user(id,password,role,username) VALUES (125,'$2a$10$YSEMa17oVO5ALwHdttK8u.iWaAHbkI9ggSDsyvOv7HqiKzi9xSGM.','manager','samata');

### CODE REVIEW

File: roomranger/frontend/src/components/roomAttendants/AddAttendant.js [hunk 1]
```diff
@@ -5,175 +5,223 @@ import AttendantListComponent from "./AttendantListComponent";
...
+                <div className="col-md-6 mb-4">
+                  <label htmlFor="notes" className="form-label ">
+                    {/* NOTES */}
+                  </label>
+                  <textarea class="form-control h-100 " id="notes" name="notes" placeholder="Notes"  onChange={(e) => handleChange(e)}></textarea>
+                </div>
+                </div >
+                <div className="container" >
+                  <label  htmlFor="workingDays" className="form-label col-md-1 checkbox-inline"><h5>Working Days</h5></label>
+                  {['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'].map((day) => (
```
Comment (LunaLiu829) [inline roomranger/frontend/src/components/roomAttendants/AddAttendant.js]: Monday to Sunday?

File: roomranger/frontend/src/App.js [hunk 2]
```diff
@@ -25,6 +25,16 @@ import Registration from './components/hompage/Registration';
 import About from './components/About';
 import React, { useState, useEffect } from 'react';
 import { Container, Row, Col } from 'react-bootstrap';
+import  AuthProvider, { useAuth } from './components/security/AuthContext';
+
+function AuthenticatedRoute({children}){
+  const authContext=useAuth()
+  if(authContext.isAuthenticated)
```
Comment (LunaLiu829) [inline roomranger/frontend/src/App.js:34]: If 
else??? Can this have 2 return?

Comment (sammu1991) [reply roomranger/frontend/src/App.js:34]: If and else does the same thing as the above code. Both conditionally render children, if the the user is authenticated.

function AuthenticatedRoute({children}){
  const authContext=useAuth()
  if(authContext.isAuthenticated){
    return children
  }
   else{
    return <Navigate to={"/login"}/>
   }

    
}

File: roomranger/frontend/src/components/roomAttendants/AddAttendant.js (hunk 1 above)
Comment (sammu1991) [reply roomranger/frontend/src/components/roomAttendants/AddAttendant.js]: Yes, you are right. I will fix this and add to the PR. Thanks.

**Review by LunaLiu829 — APPROVED** [review]

**Review by MuntoniFrancesca — APPROVED** [review]

**Review by Sonal2808 — APPROVED** [review]

**Pull request merged**